ghost/core/core/frontend/helpers/comment_count.js

```diff
@@ -3,23 +3,24 @@ const {labs} = require('../services/proxy');
 
 function commentCount(options) {
     return new SafeString(`
-       <span data-ghost-comment-count="${this.id}" style="display:none">
-           ${options.fn(this)}
-       </span>`
+       <script
```

Comment:
Any reason why we swapped to use empty script tags? To avoid styling issues?

Reply:
We didn't want to force the content to be wrapped in a tag - and a script felt like a natural thing to use here - though I think we should probably use a visual element here, so that we can have the loading state styled